### DISCUSSION TITLE
Time zone and language

### DISCUSSION THREAD

**Discussion by GnomiBerlin** [dani-garcia/vaultwarden#1324] [Q&A]
How can I set the time zone and the language in the way, that also the invitation mails are not in english?

**Comment by UnlimitedCookies** [comment]
Hey there,

I usually just mount the corresponding timezone files from the host machine into the container.
From my docker-compose:
```
 - /etc/timezone:/etc/timezone:ro 
 - /etc/localtime:/etc/localtime:ro
```

Another method is to set an environment variable. Not all containers work with this though. 
e.g. `TZ: 'Europe/Berlin'`

**Reply by nixmomo** [reply]
thats the trick! Thanks dude

**Comment by BlackDex** [comment]
Regarding the email language. That currently is English only. We do not have multi language support for this or the admin interface at the moment.

**Comment by vitorbaptista** [comment]
What about the default language? I couldn't find any env variable to set it up. Is it possible to do?

**Reply by mr-n0cturne** [reply]
I'm also interested in an variable for the default language in the web vault gui.

**Reply by BlackDex** [reply]
The web-vault uses the browser default/main language, or the user configured one.
